TalZivelin/TZivelin-HW-W01.js

```diff
@@ -0,0 +1,125 @@
+1) (Difficulty: mixed) Predict the output of each of the following expressions, and explain the rules which determine it. If the output depends on the value of x, identify the conditions when the expression will output true. Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky! Don't trust your first instinct.
+
+a) "1" == 1 - True because this does not check for the var types.
+
+b) "1" === 1 - Flase, the === checks that both type (string, NUmber) and value are identical.
+
+c) x == 'x' - If X is a number this will be true.
+
+d) x == (x+'') - True, Adding an empty string to a variable will convert it to a string which is true.
```

Comment:
1d) This isn't about the truthiness or falsiness of strings--it's actually about the equality (`==`) of the two items being evaluated. This will generally return true because an empty string won't appreciably change things where `==` is concerned, but there are a few edge cases: `NaN` is not equal to anything, including itself, and boolean values will be autoconverted to strings when added to strings, making the equality false.